pkg/util/namespace/namespace.go

```diff
@@ -23,3 +23,69 @@ func IsOpenShiftSystemNamespace(ns string) bool {
 		strings.HasPrefix(ns, "kube-") ||
 		strings.HasPrefix(ns, "openshift-")
 }
+
+// FilteredOpenShiftNamespace returns true if ns is a namespace in the defined hardcoded map
+func FilteredOpenShiftNamespace(ns string) bool {
+	filtered_namespaces := map[string]bool{
```

Comment:
typical variable usage is golang is camelcase
```suggestion
	filteredNamespaces := map[string]bool{
```

Reply:
Thanks Ben. I have updated the variable name using the pattern you suggested.